### PR TITLE
Add migrations

### PR DESCRIPTION
**Pull request by TheEyePatch**
This PR is for generating all the migrations for the application's schema. The chat_room_user model is a join table for the chat_room and devise model. The message table contains all the messages of the user which are stored in the chat_room table. The action text table is for storing the details of the tour model using the rich action text to allow the user to not only add text but to also add images in the body, to enable the user to view the images, the gems image_processing, and mini_magick is used.
The active storage model is for storing all the images and file uploads for the tours and agency credentials.

### CODE REVIEW

**Review by Eileenandrea — COMMENTED** [review]
where is the association?

**Review by Eileenandrea — CHANGES_REQUESTED** [review]
add association

**Review by Eileenandrea — APPROVED** [review]

**Review by Eileenandrea — APPROVED** [review]

File: app/models/message.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Message < ApplicationRecord
+end
```
Comment (Eileenandrea) [inline app/models/message.rb]: please add the association

File: app/models/chat_room_user.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class ChatRoomUser < ApplicationRecord
+end
```
Comment (Eileenandrea) [inline app/models/chat_room_user.rb]: please add the association

File: app/models/chat_room.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+class ChatRoom < ApplicationRecord
+end
```
Comment (Eileenandrea) [inline app/models/chat_room.rb]: please add association

**Review by Eileenandrea — CHANGES_REQUESTED** [review]

File: app/models/chat_room.rb (hunk 3 above)
Comment (TheEyePatch) [reply app/models/chat_room.rb]: Added the required associations on the last commit

File: app/models/chat_room_user.rb (hunk 2 above)
Comment (TheEyePatch) [reply app/models/chat_room_user.rb]: Added the required associations on the last commit

File: app/models/message.rb (hunk 1 above)
Comment (TheEyePatch) [reply app/models/message.rb]: Added the required associations on the last commit

**Review by morpheus17 — APPROVED** [review]
Validated and approved the changes

**Review by dlanoronald123 — APPROVED** [review]
All goods.

**Pull request merged**